.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -94,7 +94,18 @@ steps:
           - "snowy_land_longrun_gpu/*pdf"
         agents:
           slurm_gpus: 1
-          slurm_time: 24:00:00
+          slurm_time: 15:00:00
         env:
           CLIMACOMMS_DEVICE: "CUDA"
           LONGER_RUN: ""
+
+      - label: "Soil"
+        command:
```

Comment:
maybe we also need this line, which we have for the other "longer run"
```
if: build.env("LONGER_RUN") != null
```

Reply:
I renamed the group and added a label so that this mirrors the structure of our "long" runs